### ISSUE TITLE
[Question] Nested exports configuration

### ISSUE THREAD

**Issue by dmaklygin** [huozhi/bunchee#169]
Hello, sorry to disturb, I have a question about possibility to generate files according to the following structure:

```
"exports": {
    ".": {
      "import": "./dist/index.mjs",
      "require": "./dist/index.cjs"
    }
  },
```

Seems, it does not understand such structure

**Comment by huozhi** [comment]
Did you leave the command as just `"bunchee"`? For multi exports paths under "exports" field, and have a `index.<ext>` file located in the root folder. Then command `bunchee` will match the export and the input file.

Seeing https://github.com/huozhi/bunchee#multiple-exports

**Comment by huozhi** [comment]
@dmaklygin hey I feel like it could be a bug related to #171 , I've published a patch release 2.2.1. Can you try it out? hopefully it could resolve your issue, thanks
